appveyor.yml

```diff
@@ -0,0 +1,24 @@
+environment:
+  matrix:
+    - nodejs_version: "9"
+    - nodejs_version: "8"
+    - nodejs_version: "7"
+    - nodejs_version: "6"
+    - nodejs_version: "4"
```

Comment:
Can you edit this to be only 10, 9, 8, and 6? Those are our currently-supported versions.